manifests/cert.pp

```diff
@@ -41,7 +41,7 @@
   $certchainfile = "${name}_chain.crt",
   $certinterfile = "${name}_inter.crt",
   $keyfile       = "${name}.key",
-  $source        = "puppet://${::secure_server}/modules/ssldata",
```

Comment:
So does this not work because of a difference between Puppet 3.x and Puppet 4.x? Did/does the local setting of `secure_server` below override this `::secure_server`?